src/upstream.js

```diff
@@ -234,6 +234,11 @@ async function proxyResponseBodyFromUpstream(req, res, upstreamReq) {
      * client can decode the body itself.
      */
     ...(!upstreamReq.compress ? ["Content-Encoding"] : []),
+
+    /* Forward Content-Type if our response doesn't already have a preferred
+     * type set (e.g. from prior content negotiation).
+     */
+    ...(!res.get("Content-Type") ? ["Content-Type"] : []),
```

Comment:
The upstream content-type is `'application/json'` however we are converting this to `'application/json; charset=utf-8'`. In practice, I don't think clients care about charsets for JSON, but my understanding is that it shouldn't be there (see [page 11 of rfc8259](https://www.rfc-editor.org/rfc/rfc8259)).

This is automagically added by express so I'm guessing it's something we will live with:
```js
  res.set({'Content-Type': 'application/json'})
  console.log(res.get("Content-Type")) // application/json; charset=utf-8
```

but I want to point this out here as our code to copy upstream headers over to the response will not just copy them but modify them.